README.md

```
@@ -35,11 +35,33 @@ _I'd love for you to make the most of this project - it's all about learning, he
 3. **Install Required Dependencies**
 
    ```bash
+	# installs fnm (Fast Node Manager)
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)